docs/DeployConfigurations.rst

```diff
@@ -9,19 +9,27 @@ Currently, only one kind of application deployment is supported:
 Docker Deploy Daemon
 --------------------
 
-Provides the deployment of applications as Docker containers.
+Orchestrates the deployment of applications as Docker containers.
 
-There are several Docker Daemon Deploy configurations:
+.. warning::
+
+	The administrator is responsible for the security actions taken to configure the Docker daemon connected to Nextcloud.
+	For example, look at the Docker Socket Proxy protected by strict rules without root access (`like in AIO <#nextcloud-in-docker-aio-all-in-one>`_)
+
+There are several Docker Daemon Deploy configurations (example schemes):
 
 	* Nextcloud and Docker on the **same host** (via socket or port)
 	* Nextcloud on the host and Docker on a **remote** host (via port)
 	* Nextcloud and **ExApps** in the **same Docker** (via socket or port)
```

Comment:
Is this not also docker in docker?

Reply:
What exactly?

Reply:
> * Nextcloud and **ExApps** in the **same Docker** (via socket or port)

Reply:
No, they are using the same Docker, not Docker in Docker.
You can check docs with this PR and new schemes, there is `make docs` command, after which you can open `docs/_build/html/index.html` locally.

Reply:
Ah okay